docs/kToD.ts

```diff
@@ -0,0 +1,345 @@
+/*
```

Comment:
This file should not be committed. Can you add it to the gitignore so you'll notice?